src/VisualStudio/LiveShare/Impl/LoadHandler.cs

```diff
@@ -1,39 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.VisualStudio.LiveShare.LanguageServices;
-using CustomMethods = Microsoft.VisualStudio.LiveShare.LanguageServices.Protocol.CustomMethods;
-
-namespace Microsoft.VisualStudio.LanguageServices.LiveShare
-{
-    internal class LoadHandler : ILspRequestHandler, ILspRequestHandler<object, object>
```

Comment:
What was this originally used for? If old clients request it, will they gracefully handle it missing? Or do we just not care anymore? (I'm OK with not caring.)

Reply:
It used to be used to trigger loads of language servers in liveshare.  However talked to Dayton and they confirmed it's no longer used.